components/stacks-network/src/orchestrator.rs

```diff
@@ -1144,37 +1093,28 @@ amount = {}
             ));
         }
 
-        // add the 2 signers event observers
-        stacks_conf.push_str(&format!(
-            r#"
-[[events_observer]]
-endpoint = "stacks-signer-1.{}:30001"
-retry_count = 255
-include_data_events = false
```

Comment:
retry_count and include_data_events have both been removed from the observers config a while ago